galsim/config/input_cosmos.py

```diff
@@ -21,13 +21,26 @@
 
 # The COSMOSCatalog doesn't need anything special other than registration as a valid input type.
 # However, we do make a custom Loader so that we can add a logger line with some information about
-# the number of objects in the catalog that passed the initial cuts.
+# the number of objects in the catalog that passed the initial cuts and other basic catalog info.
 from .input import RegisterInputType, InputLoader
 class COSMOSLoader(InputLoader):
     def setupImage(self, cosmos_cat, config, base, logger):
         if logger:
-            logger.info("file %d: COSMOS catalog has %d total objects; %d passed initial cuts.",
-                        base['file_num'], cosmos_cat.getNTot(), cosmos_cat.getNObjects())
+            if 'input' in base:
+                if 'cosmos_catalog' in base['input']:
+                    out_str = ''
+                    if 'sample' in base['input']['cosmos_catalog'][0]:
+                        out_str += '\n  sample = %s'%base['input']['cosmos_catalog'][0]['sample']
+                    if 'dir' in base['input']['cosmos_catalog'][0]:
+                        out_str += '\n  dir = %s'%base['input']['cosmos_catalog'][0]['dir']
+                    if out_str != '':
+                        logger.warning('Using user-specified COSMOSCatalog: %s',out_str)
+            logger.warning("file %d: COSMOS catalog has %d total objects; %d passed initial cuts.",
+                           base['file_num'], cosmos_cat.getNTot(), cosmos_cat.getNObjects())
```

Comment:
Minor point, but if you're upgrading this to `warning`, then you should probably remove the `file %d:` prefix.  I used this prefix to help keep track of the (many) `info` and `debug` lines, but I think most of the `warning` logging statements don't use it.

OTOH, perhaps it would make more sense to leave this particular line as `info`.  Are these bits of information about the number of objects in the cosmos catalog things that you think people will want to see at the minimal (default) verbosity?  Seems like it's less important than the extra bits you added.